spec/services/profile_fields/import_from_csv_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.describe ProfileFields::ImportFromCsv do
+  # rubocop:disable RSpec/BeforeAfterAll
```

Comment:
The CSV import is somewhat slow since it reads from disk and writes to the DB. I didn't feel like doing this 4 times in a row, so I only import once and manually clean up after myself. I also just realized that this would make sense as a source code comment.